### PR TITLE
Fix binary parameter type

### PR DESCRIPTION
**Pull request by kuppulur**
This PR fixes the current CI issues caused due to incorrect parameter types for TFIDF and BOW methods.

- Update binary parameter type to fix the failing tests (Binary parameter for both TFIDF and BOW is a boolean parameter and not 1 or 0)

### CODE REVIEW

File: tests/test_base.py [hunk 1]
```diff
@@ -29,6 +29,10 @@ def _test_fit_transform(self, tw_model, expected, atol=1e-6):
         predicted = tw_model.fit_transform(docs)
         # torch.set_printoptions(precision=10)
         # print(predicted)
+        if not torch.allclose(predicted, expected.to(device), atol=atol):
```
Comment (skadio) [inline tests/test_base.py]: do we want the prints()? And can you add a one-liner comment on this in the code about what it does please?

File: textwiser/embeddings/random.py [hunk 2]
```diff
@@ -38,7 +38,7 @@ def random_bow(df_scale=0.1, **kwargs):
         'max_df': 1 - expon.rvs(loc=0, scale=df_scale, size=1).item(),  # max % of times a term can be found
         'min_df': expon.rvs(loc=0, scale=df_scale, size=1).item(),  # min % of times a term can be found
         # 'max_features':  # not sure how to randomly pick this
-        'binary': bernoulli.rvs(0.2, size=1).item(),  # whether to make bow binary
+        'binary': bool(bernoulli.rvs(0.2, size=1).item()),  # whether to make bow binary
```
Comment (skadio) [inline textwiser/embeddings/random.py:41]: learning q -- what's happenign here?

**Review by skadio — APPROVED** [review]
Looks good, do tests pass now? Don't you need to update Changelod per version chnage?

File: tests/test_base.py (hunk 1 above)
Comment (kuppulur) [reply tests/test_base.py]: @skadio These lines existed before, and I don't think they are necessary.

Also, this is still WIP, so please bare with me, I put a label and didn't tag you for the same reason. I will get back once, I solve the test failures. Locally, all tests pass for all environments. Somehow, they don't for 3.10 here. I am still checking.

Comment (skadio) [reply tests/test_base.py]: ah ok.. somehow it asked me for "Review". no worries

File: textwiser/embeddings/random.py (hunk 2 above)
Comment (kuppulur) [reply textwiser/embeddings/random.py:41]: So, the binary parameter in the newer sklearn models is a boolean paramater not 0/1. https://scikit-learn.org/stable/modules/generated/sklearn.feature_extraction.text.TfidfVectorizer.html

**Review by skadio — APPROVED** [review]
